libraries/AP_VisualOdom/AP_VisualOdom.cpp

```diff
@@ -114,6 +114,13 @@ const AP_Param::GroupInfo AP_VisualOdom::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_QUAL_MIN", 8, AP_VisualOdom, _quality_min, 0),
 
+    // @Param: _OPTIONS
+    // @DisplayName: Visual odometry options
+    // @Description: Visual odometry options
+    // @Bitmask: 0:Allow arming even if VO is not healthy
```

Comment:
not a blocker but we could slightly shorten the option to, "Allow arming even if unhealthy"